lib/Service/ApiService.php

```diff
@@ -215,6 +207,25 @@ public function push(Session $session, Document $document, int $version, array $
 		return new DataResponse($result);
 	}
 
+	private function addToPushQueue(Document $document, array $steps): void {
+		if ($this->queue === null || !$this->configService->isNotifyPushSyncEnabled()) {
+			return;
+		}
+
+		$sessions = $this->sessionService->getActiveSessions($document->getId());
+		$sessions = array_values(array_filter(array_unique(array_map(fn ($session): ?string => $session['userId'], $sessions))));
```

Comment:
Any particular reason to overwrite `$sessions` and not use a new variable `$userIds`? The latter would be easier to understand in my opinion.

Reply:
No reason, good call 👍